src/spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -522,37 +523,41 @@ def compute_synchrony_metrics(waveform_extractor, synchrony_sizes=(2, 4, 8), **k
     Based on concepts described in [Gruen]_
     This code was adapted from `Elephant - Electrophysiology Analysis Toolkit <https://github.com/NeuralEnsemble/elephant/blob/master/elephant/spike_train_synchrony.py#L245>`_
     """
-    assert np.all(s > 1 for s in synchrony_sizes), "Synchrony sizes must be greater than 1"
+    assert np.all([s > 1 for s in synchrony_sizes]), "Synchrony sizes must be greater than 1"
```

Comment:
Super minor but couldn't you just do
assert min(synchrony_sizes) > 1.
```
5.2 µs ± 23.8 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each) # np.all with list comprehension
```
```
135 ns ± 1.18 ns per loop (mean ± std. dev. of 7 runs, 10,000,000 loops each) # min with tuple/list
```
```
795 ns ± 2.93 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each) # min with np.array
```

But maybe I'm missing something?

Reply:
Yes! Eheheh

Reply:
Done!